pkg/common/const.go

```diff
@@ -51,3 +51,37 @@ const (
 	// TripleORolesDataFile -
 	TripleORolesDataFile = "roles_data.yaml"
 )
+
+// GetRoleGrowvolsArgs - return default growvols args for the given tripleo role name
+func GetRoleGrowvolsArgs(role string) []string {
+	if role == "Controller" {
```

Comment:
should this be a regexp to match all the different default `Controller*` roles?

Reply:
this is copied from the playbook used in a traditional director deployment so I've kept is consistent